app/liquid_tags/stackexchange_tag.rb

```diff
@@ -1,24 +1,28 @@
 class StackexchangeTag < LiquidTagBase
   PARTIAL = "liquids/stackexchange".freeze
-  # update API version here and in stackexchange_tag_spec when a new version is out
-  API_URL = "https://api.stackexchange.com/2.2/".freeze
+  REGISTRY_REGEXP = %r{https://(?<subdomain>\w*)(?:\.)?(?:stackexchange.com|stackoverflow.com)/(?<post_type>q|a|questions)/(?<id>\d{1,20})(?:/)?(?:[\w\-]*)}
```

Comment:
Refactor (blocking): `.com` will be treated as any character and then the letters `com`. The `.` needs to be escaped.

```suggestion
  REGISTRY_REGEXP = %r{https://(?<subdomain>\w*)(?:\.)?(?:stackexchange\.com|stackoverflow\.com)/(?<post_type>q|a|questions)/(?<id>\d{1,20})(?:/)?(?:[\w\-]*)}
```